tmf/org.eclipse.tracecompass.tmf.core/src/org/eclipse/tracecompass/internal/provisional/tmf/core/model/events/TmfRawEventsDataProvider.java

```diff
@@ -0,0 +1,224 @@
+/**********************************************************************
```

Comment:
The goal of the raw events provider is to get typed data out fast, please add another option for compare as instead of just equals.